src/client/xmlClient.ts

```diff
@@ -121,7 +121,7 @@ function getLanguageClientOptions(
       { scheme: 'file', language: 'dtd' },
       { scheme: 'untitled', language: 'xml' },
       { scheme: 'untitled', language: 'xsl' },
-      { scheme: 'untitled', language: 'dtd' },
+      { scheme: 'untitled', language: 'dtd' }
```

Comment:
It is because rnc have a different syntax than xml. It is not a dom document. 

If we add it in the selector lemminx will try to load rnc syntax when the file is open and will report a lot of errors.

But when xml is validated with rnc it should work. 

I think the first step is to support validation for rnc himself. It dhould not be hard since jing provide that when rnc schema is loaded. When we will do that we will need to check for all features if the open éd document is an rnc and it is case we dhould work with à rnc document instead of dom document. We could start to create a rnc document  with empty content and In the future we could implement à faut tolerant rnc parser to fill it. As the syntax is very simple I think it is doable. But I know it is not In our priority.

But having à validation for rnc should be easy to do that. I will see if I can do that In my spare time.